.circleci/config.yml

```diff
@@ -102,7 +102,8 @@ jobs:
       - *restore_repo
       - run:
           name: Test with Node 8
-          command: yarn run test
+          command: yarn test
+          command: yarn test:func
```

Comment:
I think we should keep the full name - `yarn test:functional`.

Reply:
I will change it

Reply:
Done